doc/code_snippets/snippets/config/tt.yaml

```
@@ -1,63 +1,54 @@
-tt:
-  modules:
-    # Directory where the external modules are stored.
-    directory: "modules"
+modules:
+  # Directory where the external modules are stored.
+  directory: modules
 
-  app:
-    # Directory that stores various instance runtime
-    # artifacts like console socket, PID file, etc.
-    run_dir: "var/run"
+env:
+  # Restart instance on failure.
+  restart_on_failure: false
 
-    # Directory that stores log files.
-    log_dir: var/log
+  # Directory that stores binary files.
+  bin_dir: bin
 
-    # The maximum size in MB of the log file before it gets rotated.
-    log_maxsize: 100
+  # Directory that stores Tarantool header files.
+  inc_dir: include
 
-    # The maximum number of days to retain old log files.
-    log_maxage: 8
+  # Path to directory that stores all applications.
+  # The directory can also contain symbolic links to applications.
+  instances_enabled: instances.enabled
 
-    # The maximum number of old log files to retain.
-    log_maxbackups: 10
+  # Tarantoolctl artifacts layout compatibility: if set to true tt will not create application
```

```suggestion
  # Tarantoolctl artifacts layout compatibility: if set to true, tt will not create application
```
same for 
* doc/code_snippets/snippets/replication/tt.yaml
* doc/code_snippets/snippets/sharding/tt.yaml

---

Unfortunately, cannot fix this as `tt.yaml` is generated automatically using `tt init`. So, this is a reminder that we need to review texts written by tt development team.